cli/src/action/certs.rs

```diff
@@ -231,21 +236,38 @@ fn handle_skip(
     let client_cert_path = cert_dir.join(CLIENT_CERT);
     let server_cert_path = cert_dir.join(SERVER_CERT);
     let ca_cert_path = cert_dir.join(CA_CERT);
+    #[cfg(feature = "https-certs")]
+    let rest_api_cert_path = cert_dir.join(REST_API_CERT);
 
     let client_key_path = private_cert_path.join(CLIENT_KEY);
     let server_key_path = private_cert_path.join(SERVER_KEY);
     let ca_key_path = private_cert_path.join(CA_KEY);
+    #[cfg(feature = "https-certs")]
+    let rest_api_key_path = private_cert_path.join(REST_API_KEY);
+
     let cert_path = cert_dir;
     let mut ca;
 
-    // if all exists, log existence and return
-    if client_cert_path.exists()
+    #[cfg(not(feature = "https-certs"))]
+    let all_exist = client_cert_path.exists()
+        && client_key_path.exists()
+        && server_cert_path.exists()
+        && server_key_path.exists()
+        && ca_cert_path.exists()
+        && ca_key_path.exists();
+
+    #[cfg(feature = "https-certs")]
+    let all_exist = client_cert_path.exists()
         && client_key_path.exists()
         && server_cert_path.exists()
         && server_key_path.exists()
         && ca_cert_path.exists()
         && ca_key_path.exists()
-    {
+        && rest_api_cert_path.exists()
+        && rest_api_key_path.exists();
+
+    // if all exists, log existence and return
+    if all_exist {
```

Comment:
If we hit this block, it won't print out that the REST API certs already exist. Some with the block below - which is inconsistent with how the CLI is supposed to work.

I'm not sure why we have this all_exist check at all though, since each individual check prints it out.

Reply:
Added a commit to remove this redundant check.